src/main/scala/gemmini/Controller.scala

```diff
@@ -348,12 +389,15 @@ class GemminiModule[T <: Data: Arithmetic, U <: Data, V <: Data]
   rob_completed_arb.io.in(0).valid := ex_controller.io.completed.valid
   rob_completed_arb.io.in(0).bits := ex_controller.io.completed.bits
 
-  rob_completed_arb.io.in(1) <> load_controller.io.completed
+  //rob_completed_arb.io.in(1) <> load_controller.io.completed
+  rob_completed_arb.io.in(1) <> MuxCase(load_controller(0).io.completed.valid, Seq.tabulate(num_data_controller){
```

Comment:
What happens if multiple load controllers complete at the same time?

Reply:
right, i'll increase arbiter entry

Reply:
fixed